backend/app/controllers/spree/admin/base_controller.rb

```diff
@@ -42,7 +42,7 @@ def check_alerts
         end
 
         def should_check_alerts?
-          return false if !Rails.env.production? || !Spree::Config[:check_for_spree_alerts]
+          return false if !Spree::Config[:check_for_spree_alerts]
```

Comment:
right ok, how do you feel about 
```return false if Rails.env.development? || !Spree::Config[:check_for_spree_alerts]```

My objection is to Prod-only checks, that can't be tested on the QA or staging environments (we use formal 'qa' environments here). Obviously it's not a big deal, but if there were a bug after this line of code it would cost my company a lot more money to have to debug & troubleshoot a production-only but as a result of Prod-only checks. 

Studies show that regressions caught earlier in dev cycles cost 6 times less than regressions that only show up in Production. 

It's an objection on principle, not necessarily specific to this line of code. 

Feel free to just take the other commit if you want (which only adds some stuff to the 2.3 release notes)